### ISSUE TITLE
Create user profile.

### ISSUE THREAD

**Issue by maektwain** [muellners/bounties#3]
Right now the user is restricted to a user model where email and username is stored. 

The aim 

1. We need to have user profile like , git link, resume and other important link
2. User could rate each other and should be stored in the profile.
3. Important fields like mode of work , location.
4. User related tags
5. Stores list of tasks which (recent one) user is working on.

Technical

1. Implement Model
2. Implement REST CRUD 
3. Implement UI

**Comment by xurror** [comment]
I'll start here. My UI skills are not the best but Armand or Atharva can help with that.

**Comment by maektwain** [comment]
Yeah , maybe you can implement the User profile, its wonderful lets try to close this asap.

keep in mind.

It can be mapped like this 

```
POST /user/profile
```

Should return only its relevant details. (principal details)
